vraptor-site/content/pt/docs/testando-componentes-e-controllers.html

```diff
@@ -93,6 +93,42 @@
 
 Note que qualquer chamada do tipo `result.use(...)` será ignorada.
 
+##MockSerializationResult
+
+Muito semelhante ao `MockResult` com a diferença que podemos obter o conteúdo serializado no `Result`.
```

Comment:
I'd just improve the context here, using something like: *"De forma bastante
semelhante, você pode utilizar o `MockSerializationResult` sempre que 
quiser inspecionar o conteúdo serializado no `Result`"* What do you think?